File: Liggo-api/src/Liggo.Domain/Entities/Operations/Payment.cs
using System;
using Liggo.Domain.Enums;

namespace Liggo.Domain.Entities.Operations
{
    public class Payment : BaseEntity
    {
        public Guid PlayerId { get; set; }
        public string Concept { get; set; } = string.Empty;
        public decimal Amount { get; set; }
        public DateTime Date { get; set; }
        public PaymentStatus Status { get; set; }

        // Navigation property
        public Player Player { get; set; } = null!;
    }
}

File: liggo_mobile/lib/features/payments/domain/entities/payment.dart
import 'package:equatable/equatable.dart';

class Payment extends Equatable {
  final int id;
  final String month;
  final double amount;
  final String status;
  final String date;
  final int playerId;

  const Payment({
    required this.id,
    required this.month,
    required this.amount,
    required this.status,
    required this.date,
    required this.playerId,
  });

  @override
  List<Object?> get props => [id, month, amount, status, date, playerId];
}
